### PR TITLE
Adds a floof item for Kaskreyarawkta

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Just a security HUD that looks a bit differently.

For the lovely jobs of: Head of Security, Security Pod Pilot, Warden, Security Officer, Magistrate, Internal Affairs Agent

Still need to confirm with Free if the donation went through and all.

Also comes with Human sprites.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Too OP.

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -313,6 +313,11 @@
 	icon_state = "eyepro"
 	item_state = "eyepro"
 
+/obj/item/clothing/glasses/hud/security/sunglasses/fluff/voxyHUD //LP Spartan: Kaskreyarawkta
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: don't capitalize the `HUD` portion.

Comment (Spacemanspark) [inline code/modules/customitems/item_defines.dm]: What's wrong with that, as a general question?

Comment (KasparoVy) [inline code/modules/customitems/item_defines.dm]: AFAIK it's just keeping with the general style of how things are written here, as we can see that every parent of voxyHUD all the way down the line is lowercase

Totally appearance-related

**Comment by FreeStylaLT** [comment]
👌

**Pull request merged**